app/src/componentsV2/SecondarySidebar/components/BillingTeamsNudge/BillingTeamNudge.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useCallback, useMemo, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { Button } from "lib/design-system-v2/components";
+import { MdClose } from "@react-icons/all-files/md/MdClose";
+import { useDispatch, useSelector } from "react-redux";
+import { getBillingTeamNudgeLastSeenTs, getUserAuthDetails } from "store/selectors";
+import { getCompanyNameFromEmail } from "utils/FormattingHelper";
+import { getAvailableBillingTeams, getBillingTeamMemberById } from "store/features/billing/selectors";
+import PATHS from "config/constants/sub/paths";
+import { getFunctions, httpsCallable } from "firebase/functions";
+import { actions } from "store";
+import { MdCheckCircleOutline } from "@react-icons/all-files/md/MdCheckCircleOutline";
+import { MdErrorOutline } from "@react-icons/all-files/md/MdErrorOutline";
+import { PlanStatus } from "features/settings/components/BillingTeam/types";
+import {
+  trackBillingTeamNudgeClosed,
+  trackBillingTeamNudgeRequestFailed,
+  trackBillingTeamNudgeRequestSent,
+  trackCheckoutBillingTeamNudgeClicked,
+} from "./analytics";
+import "./billingTeamNudge.scss";
+
+export const BillingTeamNudge: React.FC = () => {
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+  const user = useSelector(getUserAuthDetails);
+  const billingTeams = useSelector(getAvailableBillingTeams);
+  const teamOwnerDetails = useSelector(getBillingTeamMemberById(billingTeams[0]?.id, billingTeams[0]?.owner));
+  const billingTeamNudgeLastSeenTs = useSelector(getBillingTeamNudgeLastSeenTs);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isRequestSent, setIsRequestSent] = useState(false);
+  const [isRequestProcessed, setIsRequestProcessed] = useState(false);
+
+  const domain = getCompanyNameFromEmail(user.details?.profile?.email);
+
+  const daysSinceLastSeen = useMemo(() => {
+    const now = new Date();
+    const lastSeen = new Date(billingTeamNudgeLastSeenTs);
+    const diffTime = Math.abs(now.getTime() - lastSeen.getTime());
+    const diffDays = Math.ceil(diffTime / (1000 * 60 * 60 * 24));
+    return diffDays;
+  }, [billingTeamNudgeLastSeenTs]);
```

Comment:
can be  moved into [`DateTimeUtils.js`](https://github.com/requestly/requestly/blob/master/app/src/utils/DateTimeUtils.js) as a utility function